### PR TITLE
Poller worker minor fixes

### PR DESCRIPTION
**Pull request by claudiolor**
This PR contains some minor fixes for the poller worker:
- Rename the Poller class to Worker
- Allow to pass additional arguments to the Inventory init function
- Currently when the poller is not able to connect to all the nodes of the inventory the following error is reported:
  ```Terminating because no nodes or services found```
  This error seems to be misleading. This PR shows different errors for the nodes and the services cases.
- Add a clearer message when the host key is unknown

### CODE REVIEW

**Review by claudious96 — APPROVED** [review]

**Review by LucaNicosia — APPROVED** [review]

File: suzieq/poller/worker/poller.py [hunk 1]
```diff
@@ -153,8 +178,11 @@ def _init_inventory(self, userargs, cfg):
             'ssh_config_file': userargs.ssh_config_file,
         }
 
+        if addnl_args:
+            inventory_args.update(addnl_args)
+
         # Retrieve the specific inventory source to use
-        inv_types = Inventory.get_plugins()
+        inv_types = self._get_inventory_classes()
```
Comment (ddutt) [inline suzieq/poller/worker/poller.py:197]: Whats the motivation to add a new function and change the name?

File: suzieq/poller/worker/nodes/node.py [hunk 2]
```diff
@@ -365,8 +365,15 @@ async def _init_ssh(self, init_boot_time=True, rel_lock=True) -> None:
                 if self.sigend:
                     await self._terminate()
                     return
-                self.logger.error("ERROR: Unable to connect to "
-                                  f"{self.address}:{self.port}, {e}")
+                if isinstance(e, asyncssh.HostKeyNotVerifiable):
+                    self.logger.error(
+                        f'Unable to connect to {self.address}: {self.port}, '
+                        'host key is unknown. If you do not need to verify '
+                        'the host identity, add "ignore-known-host: True" in '
```
Comment (ddutt) [inline suzieq/poller/worker/nodes/node.py:372]: The keyword is "ignore-known-hosts", not "ignore-known-host".

**Review by ddutt — CHANGES_REQUESTED** [review]
One MUST FIX before commit and the others are just comments or q.

File: suzieq/poller/worker/poller.py (hunk 1 above)
Comment (ddutt) [reply suzieq/poller/worker/poller.py:197]: Yes, it does, thank you

**Pull request merged**